### DISCUSSION TITLE
How to increase power consumption after losing focus on the wbview2 window

### DISCUSSION THREAD

**Discussion by duwenlong2** [MicrosoftEdge/WebView2Feedback#4178] [Q&A]
I used WPF to nest webview2 controls, and webview2 loads a local webpage that will call the camera to display on the interface for some calculations.
When my WPF program loses focus, the page loaded by webview2 becomes sluggish after a period of running, and the video feedback results become sluggish. Because I only have one page, I want to keep the webview2 control or this page running at high performance when I lose focus.
Is there a place where I can set this function?

**Comment by victorhuangwq** [comment]
@lflores-ms could you take a look at this question? My understanding as of now is that you can't do that.

**Reply by duwenlong2** [reply]
Thank you, I am also looking for a solution. If we have ideas, we can discuss them. After testing, I found that the laptop has a higher delay when it is not plugged into an external power source and in an energy-saving state. It takes 1-2 seconds to process one frame of image.

**Reply by lflores-ms** [reply]
Hi @duwenlong2,

Are there timers driving your app? If there are, the behavior you're describing sounds like script timers getting throttled. When the WPF WebView2 control is hidden, its underlying controller `IsVisible` property will be set false and allow the page to move to background state. As of now, we don't have support to make changes on this behavior, but we have an API in the works that would let you do this: [API Review: Throttling Control - Script](https://github.com/MicrosoftEdge/WebView2Announcements/issues/80)
